pkg/storage/storage_fs.go

```diff
@@ -34,7 +34,7 @@ const (
 	// BlobUploadDir defines the upload directory for blob uploads.
 	BlobUploadDir = ".uploads"
 	SchemaVersion = 2
-	gcDelay       = 1 * time.Hour
+	gcDelay       = 1 * time.Nanosecond
```

Comment:
This can't be right! 
You can add another delay timer for this.

Reply:
The problem is that I have to use the function ifOlderThan(), which is non exported, so I can't use it as I would wish. 
So this was just a temporary workaround to check if my code actually deletes blobs.